src/block/parser.rs

```diff
@@ -237,13 +237,13 @@ impl BlockParser {
         Ok(next_block)
     }
 
-    fn empty(blocks_dir: &Path, paths: &Vec<PathBuf>) -> Self {
+    fn empty(blocks_dir: &Path, paths: &[PathBuf]) -> Self {
         Self {
             num_canonical: 0,
             total_num_blocks: paths.len() as u32,
             blocks_dir: blocks_dir.to_path_buf(),
             canonical_paths: vec![].into_iter(),
-            recent_paths: paths.clone().into_iter(),
+            recent_paths: paths.to_owned().into_iter(),
```

Comment:
Is this change related to the Rust minor version change?

Reply:
See previous comment